lib/ddtrace/contrib/rails/core_extensions.rb

```diff
@@ -150,8 +150,13 @@ def process_action_with_datadog(*args)
           # signals; it propagates the request span so that it can be finished
           # no matter what
           payload = {
-            controller: self.class.name,
+            controller: self.class,
```

Comment:
can you provide more details about this change?

Reply:
It was only passing a string of the class name before. We need the actual constant now, so we can actually do type checking.

Reply:
changing that doesn't have any side effect for the rest of the code, right?

Reply:
Nope, it was being used to tag the name on the span. Since `"#{class}"` is the same thing as `class.name`, it has no change.